test/role.test.js

```diff
@@ -363,7 +363,7 @@ describe('role model', function() {
     });
   });
 
-  it('should support owner role resolver', function(done) {
+  it('should support legacy owner role resolver', function(done) {
```

Comment:
`it(supports legacy...)` - see http://loopback.io/doc/en/contrib/style-guide.html